lib/sport_type.rb

```diff
@@ -1,44 +1,45 @@
 # frozen_string_literal: true
 
 class SportType
-  def self.id_for(name: nil)
-    return 5 unless name
-
-    SPORT_TYPE_MAPPING[name]
+  def self.unified(name:)
+    SPORT_TYPE_UNIFIED[name] || name
   end
 
-  def self.name_for(id: nil)
+  def self.name_for_runtastic_id(id: nil)
     return 'other' unless id
 
-    SPORT_TYPE_MAPPING.key(id)
+    SPORT_TYPE_MAPPING.fetch(id)
```

Comment:
`RUNTASTIC_SPORT_TYPE_MAPPING.fetch(id)`

Reply:
Fixed + added tests for runtastic-json parser